--- a/configure	2020-04-20 22:48:22.000000000 +0200
+++ b/configure	2020-04-20 22:49:07.000000000 +0200
@@ -4668,8 +4668,7 @@
    }
 
   if ((gtk_major_version != $gtk_config_major_version) ||
-      (gtk_minor_version != $gtk_config_minor_version) ||
-      (gtk_micro_version != $gtk_config_micro_version))
+      (gtk_minor_version != $gtk_config_minor_version))
     {
       printf("\n*** 'pkg-config --modversion gtk+-3.0' returned %d.%d.%d, but GTK+ (%d.%d.%d)\n",
              $gtk_config_major_version, $gtk_config_minor_version, $gtk_config_micro_version,
@@ -4683,8 +4682,7 @@
       printf("*** to point to the correct configuration files\n");
     }
   else if ((gtk_major_version != GTK_MAJOR_VERSION) ||
-	   (gtk_minor_version != GTK_MINOR_VERSION) ||
-           (gtk_micro_version != GTK_MICRO_VERSION))
+	   (gtk_minor_version != GTK_MINOR_VERSION))
     {
       printf("*** GTK+ header files (version %d.%d.%d) do not match\n",
 	     GTK_MAJOR_VERSION, GTK_MINOR_VERSION, GTK_MICRO_VERSION);
